modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -0,0 +1,860 @@
+----
+CREATE OR REPLACE USER name
+  SET [PLAINTEXT \| ENCRYPTED] PASSWORD 'password'
+  [[SET PASSWORD] CHANGE [NOT] REQUIRED]
+  [SET STATUS {ACTIVE \| SUSPENDED}]
+  [SET HOME DATABASE name]
+----
+
+| Description
+a|
+Creates a new user, or if a user with the same name exists, replace it.
+
+For more information, see xref:authentication-authorization/manage-users.adoc#access-control-create-users[Creating users].
+
+| Required privilege
+a|
+[source, privilege, role="noheader"]
+----
+GRANT CREATE USER
+----
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[DBMS USER MANAGEMENT privileges])
+
+
+[source, privilege, role="noheader"]
+----
+GRANT DROP USER
+----
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[DBMS USER MANAGEMENT privileges])
+
+|===
+
+[cols="<15s,<85"]
+|===
+| Command
+m| RENAME USER
+
+| Syntax
+a|
+[source, syntax, role="noheader"]
+----
+RENAME USER name [IF EXISTS] TO otherName
+----
+
+| Description
+a|
+Changes the name of a user.
+
+For more information, see xref:authentication-authorization/manage-users.adoc#access-control-rename-users[Renaming users].
+
+| Required privilege
+a|
+[source, privilege, role="noheader"]
+----
+GRANT RENAME USER
+----
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[DBMS USER MANAGEMENT privileges])
+
+|===
+
+[cols="<15s,<85"]
+|===
+| Command
+m| ALTER USER
+
+| Syntax
+a|
+[source, syntax, role="noheader"]
+----
+ALTER USER name [IF EXISTS]
+  [SET [PLAINTEXT \| ENCRYPTED] PASSWORD 'password']
+  [[SET PASSWORD] CHANGE [NOT] REQUIRED]
+  [SET STATUS {ACTIVE \| SUSPENDED} ]
+  [SET HOME DATABASE name]
+  [REMOVE HOME DATABASE]
+----
+
+| Description
+a|
+Modifies the settings for an existing user.
+At least one `SET` or `REMOVE` clause is required.
+`SET` and `REMOVE` clauses cannot be combined in the same command.
+
+For more information, see xref:authentication-authorization/manage-users.adoc#access-control-alter-users[Modifying users].
+
+| Required privilege
+a|
+[source, privilege, role="noheader"]
+----
+GRANT SET PASSWORD
+----
+
+[source, privilege, role="noheader"
+----
+GRANT SET USER STATUS
+----
+
+[source, privilege, role="noheader"]
+----
+GRANT SET USER HOME DATABASE
+----
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-user-management[DBMS USER MANAGEMENT privileges])
+
+|===
+
+
+[cols="<15s,<85"]
+|===
+
+| Command
+m| ALTER CURRENT USER SET PASSWORD
+
+| Syntax
+a|
+[source, syntax, role="noheader"]
+----
+ALTER CURRENT USER SET PASSWORD FROM 'oldPassword' TO 'newPassword'
+----
+
+| Description
+a|
+Changes the current user's password.
+
+For more information, see xref:authentication-authorization/manage-users.adoc#access-control-alter-password[Changing the current user's password].
```

Comment:
No idea, as I said, I haven't updated those pages, just the links.

Reply:
I'll assume it will be updated once additional PRs are ported afterwards but will leave the comment open to remember to check that if that's ok with you?